### PR TITLE
Readme cleanup

### PR DESCRIPTION
**Pull request by naclander**
* Corrected rst syntax
* Grammar cleanup
* Removed verbose debugging option from examples

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -29,9 +29,10 @@ common case:
 Client side Requirements
 ------------------------
 
-- sudo, or logged in as root on your client machine.
+- sudo, or root access on your client machine.
   (The server doesn't need admin access.)
 - Python 2.7 or Python 3.5.
+- For Python 2.7, you need PyXAPI, available `here <http://www.pps.univ-paris-diderot.fr/~ylg/PyXAPI/>`_
```
Comment (brianmay) [inline README.rst]: PyXAPI is only required for Python 2.7 *and* TProxy with UDP or DNS enabled. That is why it was below the UDP or DNS paragraph. I realize this was not clear however... Maybe it should be included in that paragraph.

File: README.rst [hunk 2]
```diff
@@ -63,13 +64,10 @@ Additional information for TPROXY
 ---------------------------------
 TPROXY is the only method that supports full support of IPv6 and UDP.
 
-Full UDP or DNS support with the TPROXY method requires the `recvmsg()`
+Full UDP or DNS support with the TPROXY method requires the ``recvmsg()``
 syscall. This is not available in Python 2.7, however is in Python 3.5 and
 later.
```
Comment (brianmay) [inline README.rst]: This is the paragraph I was talking about above. Only need PyXAPI for this. Actually I haven't tested PyXAPI for sometime, so might be worth just dropping all references to it - now that we know Python 3.5 has all the support required.

**Comment by brianmay** [comment]
All other changes look good to me.

File: README.rst (hunk 1 above)
Comment (naclander) [inline README.rst]: I suppose the best place for this could be in the table in the tproxy Requirements section.

**Comment by brianmay** [comment]
Merged everything except the PyXAPI change.

**Comment by brianmay** [comment]
Made some changes for PyXAPI. Closing.

**Pull request closed without merging**